arrow/src/ffi_stream.rs

```
@@ -358,9 +358,8 @@ impl Iterator for ArrowArrayStreamReader {
             }
 
             let data = unsafe {
-                from_ffi_and_data_type(array, DataType::Struct(self.schema().fields().clone()))
-            }
-            .ok()?;
+                from_ffi_and_data_type(array, DataType::Struct(self.schema().fields().clone()))?
+            };
```

```
error[E0277]: the `?` operator can only be used on `Option`s, not `Result`s, in a method that returns `Option`
```

Oh right, I think one reason to suppress error is because that too.

---

Oh, I don't open the file to look at, actually `Item = Result<RecordBatch>`. 👍